### DISCUSSION TITLE
freetz linux

### DISCUSSION THREAD

**Discussion by Monarch73** [Freetz-NG/freetz-ng#571] [Allgemein]
Ich bin davon ausgegangen, das es sich bei dem Freetz-Linux 2.0 um ein Image handelt, in dem alle Abhängigkeiten installiert sind, um ein image zu bauen. Tatsächlich fehlen aber für ein "make" libs, die ziemlich basic sind (libc6-dev) usw....

Wofür ist dieses Image denn gedacht, wenn die da die grundlegendsten Sachen fehlen?

**Comment by fda77** [comment]
Hallo,
Gismotro ist ein bekennender Apter und vollzieht sein Begehr täglich. Um es anderen auch nahezubringen hatte er eine simple Idee: Ein Linux nehmen und alles entfernen was nur möglich ist! So kann jeder das tun was er so liebt: apten! Und das kannst du nun auch!!1 Dazu den Scnapshot der VM 1-3 mal täglich zurücksetzen. So kannst auch du ein Aptguru werden! Bei Beschwerden gehts ins IPPF oder DEB wo Gismotro öfter ist, ich find den passenden Thread jetzt nicht (hab auch gar nicht gesucht)

**Reply by Monarch73** [reply]
was bitte ist mit "apter" gemeint? Jemand, der gerne apt benutzt?

Wenn es darum geht, ein minimalistisches Linux zu haben, hat "gismotro" aber kräftig daneben gegriffen: ein debootstrap führt zu einer minimalen Ubuntu-Installation, bei der das Image wenige 100MB  (ca.600-700MB) groß werden würde. Das Freetz-Linux image hat knapp 4GB!

**Comment by fda77** [comment]
Genau er hängt den Aptern an. Wahrscheinlich konnte er es nicht besser machen, da die Apter im heiligen Krieg gegen die Dnfer sind. Die Fedora User müssen endlich zum einzig richtigen Paketmanager bekehr werden. Wo käme man hin wenn jeder einen anderen benutzt?
Vor allem die Sexuelle Enthaltsamkeit der Dnfer gefällt den Aptern nicht. Das namensgebende DNF das "Dandified Yum" bedeuten soll hat nämlich einen anderen Ursprung und ist ein Vorschschrift! Das wird aber geheimgehalten um neue Mitglieder nicht abzuchrecken!

**Reply by Monarch73** [reply]
Sagt doch einfach, das man sich das VM-Image schenken kann und gut ist....

**Comment by fda77** [comment]
Nö, viel zu langweilig.

**Comment by fda77** [comment]
Wie gesagt.. frag Gismotro in einem der beiden Foren.
Ich benutze das nicht und kann dazu nichts `sinvolles `sagen

**Reply by Monarch73** [reply]
Wenn ich die PREREQUISITES.md richtig interpretiere, ist fedora die distro, wo man am wenigsten händisch nachziehen muss. 

Stimmt das so ungefähr?

**Comment by fda77** [comment]
Man würde es besser so sagen: Ich benutze Fedora selbst, daher sind die prerequisites da am genauesten
Ubuntu auch aber nur seltzen "etwas". Eigentlich sollte jeder da was ändern wenn etwas nicht passt. Das klappt aber nicht
Theoretisch kann man jedes Linux nutzen, solang man alles nötige installiert hat.
Heute kamen zb noch ein paar neue prereq. dazu, die sind bestimmt noch in keiner VM

**Reply by Monarch73** [reply]
Im prinzip ist mir die distro wurscht, aber ich habe keine Lust auf Ewigkeiten dauernde bastel-orgien. 

Ich schau mal, ob ich mit einer minimalinstallation eines fedora-server klar komme und versuche mal ein Fritzbox 4060-image zu bauen.

**Comment by fda77** [comment]
Am besten bleibt man für immer auf der gleichen Revision, so ändern sich die prerequsistes nicht

**Reply by Monarch73** [reply]
Ich habe eine Fedora-Server 36 Minimalinstallation vorgenommen und die in der prerequisistes definierten pakete nachinstalliert. Funktioniert problemlos.

Wie sieht es denn mit zukünftigen Firmware-Updates aus? Muss das jetzt immer manuell über freetz laufen?
